ComfyUI/custom_nodes/luma_photon_node/luma_photon_node.py

```
@@ -0,0 +1,238 @@
+from __future__ import annotations
+import torch
+import numpy as np
+from PIL import Image
+import os
+import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 5

Remove definition: `os`

(F811)